backend/src/Controllers/voterRolls.controller.ts

```diff
@@ -60,7 +67,42 @@ const addElectionRoll = async (req: any, res: any, next: any) => {
         return responseErr(res, req, 500, msg);
     }
 }
-
+const registerVoter = async (req: any, res: any, next: any) => {
```

Comment:
In the future I could see a bulkRegisterVoter being handy (for example, if IPO wanted to use this API to register their full voter list)

Reply:
We sort of have that with adding the voter id list. Registering marks them as registered which would then need approval. But if IPO is adding them then they would already be approved?